sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1851,7 +1835,7 @@ SUBROUTINE INITPOST_NETCDF(ncid3d)
 
 
 ! instantaneous 3D cloud fraction
-      VarName='cldfra'
+      VarName='cldfra_bl'
```

Comment:
@EricJames-NOAA This change would have impact on EMC RRFS which uses "cldfra" from model output. I invite @ericaligo-NOAA and @EricRogers-NOAA from EMC side for their comments.

Reply:
Yes, cldfra is the 3D instantaneous cloud fractions that can be used with any microphysics scheme.  A separate PR will be created soon to replace this name with cld_amt in order for it to be used for both regional and global applications.  cldfra is currently used with the RRFS.

Reply:
@EricJames-NOAA Could you revert this change? @ericaligo-NOAA would submit another PR at UPP and UFS for unifying 3D instant cloud fraction name as cld_amt?

Reply:
I will revert this change.  Thanks @WenMeng-NOAA and @ericaligo-NOAA.